data/hai/unfettered 1 alphas.txt

```diff
@@ -0,0 +1,1528 @@
+			label beforeugo
+			`	"You should know that examining these specs will take me some time, so it could be a while until I employ you again," she says over her head whilst leaving.`
+			choice
+				`	"Wait, what are the 'unfettered humans'?"`
+					to display
+						not "unfettered: it's the Alphas!"
+				`	(I'll find it out on my own.)`
+					decline
+			`	She stops on her track, before turning around partially to answer, "We use 'unfettered' to describe groups that fight for what they believe in. That includes the anarchists organizations you refer to as pirates, but not only." She is clearly unwilling to tell you more than this and goes back on her way.`
+
+
+
+# SPOILERS: the disaster was caused by the Alphas as a threat (and also because the Unfettered allowed you to see them)
+# this also is to increase tensions and to make Unfettered attack Wanderers faster.
+mission "<uhai> 6 A: Loyal Escort"
+	landing
+	name `Escort a Hai convoy`
+	description `Escort a Hai convoy from <stopovers> to <destination> before <date>.`
+	source "Warfeed"
+	stopover "Greenbloom"
+	deadline 20
+	to offer
+		has "<uhai> 0: Bring Hai: done"
+		has "Unfettered Jump Drive 2: offered"
+		or
+			has "<uhai> 5: Scan Auxiliary: done"
+			has "Unfettered: Alpha Supplies: Coming: offered"
+			has "Unfettered: Alpha Supplies: Leaving: offered"
+		has "midas: tracker relay"
+	on offer
+		conversation
+			`As you leave your ship, an Unfettered Chief approaches you. After some time you finally recognize Sookootow, the one you helped earlier with moving people and medical supplies. You have not yet seen him so worried, nor moving at such speed.`
+			`	"<first>! I asked to be notified of your arrival as soon as possible, as we need your services. A fleet of Hai needs your immediate assistance escorting them through Hai space, from <stopovers>. They may not be Unfettered, but their hearts beat true still. I would blame myself if anything was to happen to them, and I could have done anything to prevent it."`
+			choice
+				`	"Why do you need my help specifically?"`
+				`	"What are you even doing here?"`
+					goto purposehere
+				`	"I'm busy right now, sorry."`
+					goto busy
+			`	"It is not that I do not trust the Hai not to fire on their own people, but a misunderstanding happens quickly. Someone of your expertise would be helpful, and I cannot simply trust any Unfettered captain with this, as they could easily be provoked into making a fatal mistake. I hope, though, that you will be able to make a rational decision.`
+			`	He gets closer to you, and looks around to make sure nobody listens, except for one Unfettered, moving as his shadow, that you assume to be his bodygard. "I also need your presence because I want someone that is quick to respond to danger, and with the right tools. Hearing of your duel with a fellow Unfettered captain, you strike me as such a person. We are worried that another party may interfere with our plans."`
```

Comment:
```suggestion
			`	He gets closer to you, and looks around to make sure nobody listens, except for one Unfettered, moving as his shadow, that you assume to be his bodyguard. "I also need your presence because I want someone that is quick to respond to danger, and with the right tools. Hearing of your duel with a fellow Unfettered captain, you strike me as such a person. We are worried that another party may interfere with our plans."`
```